test/data/EnumerableMap.ts

```diff
@@ -0,0 +1,415 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { bnToAddress } from '@solidstate/library';
+import {
+  EnumerableMapAddressToAddressMock,
+  EnumerableMapAddressToAddressMock__factory,
+  EnumerableMapUintToAddressMock,
+  EnumerableMapUintToAddressMock__factory,
+} from '@solidstate/typechain-types';
+import { expect } from 'chai';
+import { BigNumber } from 'ethers';
+import { ethers } from 'hardhat';
+
+describe('EnumerableMap', () => {
+  describe('AddressToAddressMap', async () => {
+    let instance: EnumerableMapAddressToAddressMock;
+    let deployer: SignerWithAddress;
+
+    beforeEach(async () => {
+      [deployer] = await ethers.getSigners();
+      instance = await new EnumerableMapAddressToAddressMock__factory(
+        deployer,
+      ).deploy();
+    });
+
+    describe('__internal', () => {
+      const addressOne = bnToAddress(BigNumber.from('100'));
+      const addressTwo = bnToAddress(BigNumber.from('200'));
+      const addressThree = bnToAddress(BigNumber.from('300'));
+      const addressFour = bnToAddress(BigNumber.from('400'));
+      const addressFive = bnToAddress(BigNumber.from('500'));
+      const addressSix = bnToAddress(BigNumber.from('600'));
+
+      describe('#at(uint256)', () => {
+        it('returns value coresponding to index provided', async () => {
+          await instance['set(address,address)'](addressOne, addressFour);
+          await instance['set(address,address)'](addressTwo, addressFive);
+          await instance['set(address,address)'](addressThree, addressSix);
```

Comment:
In several places you have multiple similar calls.  I don't think it adds much value.  One is enough.